File: Backend/Framework.Core/Common/Pagination.cs
#region Using ...
#endregion

/*
 
 
 */
namespace Framework.Core.Common
{
    /// <summary>
    /// 
    /// </summary>
    public class Pagination
	{
		#region Properties
		public int? PageIndex { get; set; } = 0;
		public int? PageSize { get; set; } = 10;
		public long? TotalCount { get; set; }
		public bool GetTotalCount { get; set; } = true;
		#endregion
	}
}

File: Frontend/src/app/modules/dashboard/models/pagination.ts
export class Pagination {
  pageIndex: number;
  pageSize: number;
  totalCount: number;
  getTotalCount: boolean;

  constructor(
    pageIndex: number = 0,
    pageSize: number = 10,
    totalCount: number = 0,
    getTotalCount: boolean = true
  ) {
    this.pageIndex = pageIndex;
    this.pageSize = pageSize;
    this.totalCount = totalCount;
    this.getTotalCount = getTotalCount;
  }
}
